addon/bootstrap.js

```diff
@@ -51,8 +52,11 @@ const prefObserver = {
   }
 };
 
+let localePref = (prefs.getPrefType(LOCALE_PREF) && prefs.getCharPref(LOCALE_PREF)) || "en-US";
+
 function startup(data, reason) { // eslint-disable-line no-unused-vars
   console.log("bootstrap startup called");
+  Cu.import("chrome://screenshots/content/localizedTooltips.jsm");
```

Comment:
I *think* it should work okay, since this just lazy-loads the module. I can try both in my local Firefox and see.